db/main/structure.sql

```diff
@@ -1549,6 +1581,7 @@ CREATE TABLE public.deleted_jobs (
     com_id integer,
     config_id integer,
     restarted_at timestamp without time zone,
+    restarted_by integer DEFAULT NULL,
```

Comment:
is there also some migration adding restarted_by to add, or is it already present?

Reply:
Yes, [here](https://github.com/travis-ci/travis-migrations/blob/master/db/main/migrate/20220621151453_add_column_restarted_by_to_table_jobs.rb).